File: abc/abc218/b.cpp
// https://atcoder.jp/contests/abc218/submissions/25760451

#include <bits/stdc++.h>
using namespace std;

int main() {
  int N;
  for(int i = 0; i < 26; i++) {
    cin >> N;
    N += 96;
    cout << (char)N;
  }
}

File: abc/abc218/b.cs
// https://atcoder.jp/contests/abc218/submissions/46938169

class A
{
  static void Main()
  {
    string[] t = Console.ReadLine().Split(" ");
    int[] p = new int[26];
    for(int i = 0; i < 26; i++)
    {
      p[i] = int.Parse(t[i]);
      Console.Write((char)(96 + p[i]));
    }
  }
}

